models/repo/repo.go

```diff
@@ -277,7 +277,7 @@ func (repo *Repository) AfterLoad() {
 	repo.NumOpenProjects = repo.NumProjects - repo.NumClosedProjects
 	repo.NumOpenActionRuns = repo.NumActionRuns - repo.NumClosedActionRuns
 
-	repo.ObjectFormat = git.ObjectFormatFromID(git.Sha1)
+	repo.ObjectFormatName = "sha1"
```

Comment:
`Sha1ObjectFormat.Name()` ?

Reply:
I would like not to reference git module from this package.

Reply:
Leave a comment?